UsageCollectorWorkerService/Services/LowLevelCollecting/LowLevelCollectingService.cs

```diff
@@ -23,10 +23,10 @@ public async Task<byte> GetCpuUsageInPercent()
         string[] lines = output.Split("\n");
         string[] memory = lines[3].Split(" ", StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Normally you don't want "magical values" (like 3 in this case) anywhere. You should put them into variables and use these. But I'm not sure if in this case this makes sense. Maybe a comment with reasoning why 3 is used here might be better, but I don't know the context here. If the comment won't bring any value just ignore this suggestion